### ISSUE TITLE
Use Github Bot for commit linting

### ISSUE THREAD

**Issue by petermetz** [hyperledger-labs/blockchain-integration-framework#53]
This or something like it:
https://github.com/z0al/commitlint-bot

**Comment by jonathan-m-hamilton** [comment]
Can I take this one? ^_^

**Comment by petermetz** [comment]
Sure thing 👍

**Comment by petermetz** [comment]
Just saw this one, also highly recommended:
https://help.github.com/en/github/managing-security-vulnerabilities/configuring-automated-security-updates

**Comment by jonathan-m-hamilton** [comment]
I don't have sufficient access to install on this Lab, but I have requested.  We would also need elevated access to make mandatory for commits.  I have installed in my own fork to try out.  @tkuhrt 

![image](https://user-images.githubusercontent.com/33758641/70765336-1d8a8e00-1d20-11ea-882f-2a343b7329ef.png)


![image](https://user-images.githubusercontent.com/33758641/70765278-e1efc400-1d1f-11ea-87f1-ab695a012acb.png)
